tests/manage/mcg/test_pv_pool.py

```diff
@@ -221,6 +228,14 @@ def test_pvpool_resource_modifications(
             label=pv_pod_label, namespace=config.ENV_DATA["cluster_namespace"]
         ):
             pv_pod_obj.append(Pod(**pod))
+
+        # verify there is no dummy access_key, secret_key messages are seen in the pv pool pod
+        for pod in pv_pod_obj:
+            pod_logs = get_pod_logs(pod_name=pod.name)
+            assert ("access_key:" not in pod_logs) or (
+                "secret_key:" not in pod_logs
+            ), f"access_key or secret_key are logged in the pv pool pod {pod.name} logs"
```

Comment:
```suggestion
            assert ("access_key:" not in pod_logs) and (
                "secret_key:" not in pod_logs
            ), f"access_key or secret_key are logged in the pv pool pod {pod.name} logs"
```
We need to assert that both target strings are missing from the logs, and this currently evaluates as True even if one of these are present:

```
$ python
Python 3.8.16 (default, Dec  7 2022, 01:27:54) 
[Clang 14.0.0 (clang-1400.0.29.202)] on darwin
Type "help", "copyright", "credits" or "license" for more information.

>>> pod_logs = "secret_key:"
>>> ("access_key:" not in pod_logs) or ("secret_key:" not in pod_logs)
True

>>> pod_logs = "access_key: secret_key:"
>>> ("access_key:" not in pod_logs) or ("secret_key:" not in pod_logs)
False
```

Reply:
@sagihirshfeld I guess we need to assert when either of them present also. since it is not expected for either access_key or secret_key to be logged in the any sort of logs, its a security vulnerability. Hence OR is better than AND